pkg/apiserver/config.go

```diff
@@ -16,6 +16,44 @@ limitations under the License.
 
 package apiserver
 
+import (
+	"k8s.io/apimachinery/pkg/apimachinery/announced"
+	"k8s.io/apimachinery/pkg/apimachinery/registered"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"k8s.io/apimachinery/pkg/runtime/serializer"
+
+	"github.com/kubernetes-incubator/service-catalog/pkg/apis/servicecatalog/install"
+)
+
+var (
+	groupFactoryRegistry = make(announced.APIGroupFactoryRegistry)
+	registry             = registered.NewOrDie("")
+	// Scheme for API object types
+	Scheme = runtime.NewScheme()
+	// Codecs for creating a server config
+	Codecs = serializer.NewCodecFactory(Scheme)
+)
+
+func init() {
```

Comment:
ask @deads2k :) https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/sample-apiserver/pkg/apiserver/apiserver.go#L44

I don't remember why I copied this, presumably something didn't work without that but I'll double check.